refinery/core/tests.py

```diff
@@ -2413,6 +2415,16 @@ def setUp(self):
         self.good_dataset = create_dataset_with_necessary_models()
         self.bad_dataset = DataSet.objects.create()
 
+    def test_prepare(self):
+        data = self.dataset_index.prepare(self.good_dataset)
+        self.assertRegexpMatches(
+            data['text'],
+            re.compile(
+                r'AnnotatedNode-\d.*AnnotatedNode-\d',
```

Comment:
This depends on the names the factory fills in as a default. Widely separated dependencies on the same string aren't reassuring.